.github/workflows/docker.yaml

```diff
@@ -15,6 +15,11 @@ on:
         description: Image base name, without tag.
         type: string
         required: true
+      additional_release_image_name:
```

Comment:
Seems like "additional_release_repo_name" might be better? The other image name doesn't include the repo, just the base name of the image. Might be better for this to be a repo name that is prepended to the image name to push to the second place.

Reply:
> The other image name doesn't include the repo, just the base name of the image

That's not true. For example. `usage_metrics` has 
```
image_name: gcr.io/ironcore-dev-1/usage-metrics
```

Reply:
Ah - sorry - I thought that got built up somewhere. I guess I didn't look hard enough at where it substituted ironcore-dev-1 for ironcore-images (or vice versa).